src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.css

```diff
@@ -7,11 +7,12 @@
         "head"
         "toolbar"
         "main";
+    gap: calc(var(--design-unit) * 2px);
 }
 
 ::deep.resource-logs-layout > h1 {
     grid-area: head;
-    padding: calc(var(--design-unit) * 1px);
+    padding: calc(var(--design-unit) * 1px) calc(var(--design-unit) * 2px);
```

Comment:
This is the same across the pages. Add a style to the global CSS file and then add class to these titles? That would reduce the chance of the spacing from going out of sync the next time a change happens.

Reply:
Good call. Done.